### DISCUSSION TITLE
Is docfx.console nuget dead?

### DISCUSSION THREAD

**Discussion by famda** [dotnet/docfx#9103] [General]
Hey,
Is the nuget package docfx.console dead? 
The latest version of the package is 2.59.4.
I'm having some issues with the modern template not being generated.

Thanks ;)

**Comment by KalleOlaviNiemitalo** [comment]
Yes it was replaced with the "[docfx](https://www.nuget.org/packages/docfx)" .NET tool package.  It should be marked deprecated in NuGet; I have filed <https://github.com/dotnet/docfx/issues/8932> for that.

**Comment by famda** [comment]
Thanks for your answer @KalleOlaviNiemitalo 
Is there another nuget that we can reference that we can configure on the csproj and run with the build?
The same way the console package was working (MSBUILD)? 

Like that we don't need to install tooling anywhere on our dev environments.

**Comment by KalleOlaviNiemitalo** [comment]
AFAIK, DocFX no longer provides a package that would integrate with MSBuild and automatically build the docs when you build a C# or VB project.

**Reply by famda** [reply]
For some use cases, it would be nice to have this.
I'm making some experiments that generate documentation on build and render it in a blazor app.
The integration with MSbuild the same way the console package was is very nice because it doesn't require the installation of the tool and it just works for this use case. 😀

**Reply by lanthonyneville** [reply]
After struggling with this myself, I wrote up how I did this here: https://github.com/dotnet/docfx/discussions/8605
